packages/sdk/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@saagie/sdk",
-  "version": "0.12.4",
+  "version": "0.12.5-alpha.4+257eb5d",
```

Comment:
Should we modify this version to a stable one ? Or is it automatically modified by the GH Action ?

Reply:
It probably have been automatically updated by my test of the publication. I have reverted it. It is indeed updated by the CI.